aspnetcore-features/HowToReadConnectionStringsInDotNet/HowToReadConnectionStringsInDotNet/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+
+namespace HowToReadConnectionStringsInDotNet.Controllers
```

Comment:
Can we use file-scoped namespaces? It looks better in my mind since we can just remove one level of indentation from our files :)